analytics.py

```diff
@@ -0,0 +1,98 @@
+from .utils import *
```

Comment:
be very careful using `*` imports - they work but are discouraged because of namespace pollution.  Not a big deal for an assignment like this, but image working on a much larger system that leverages external libraries.